pkg/graveler/ref/manager.go

```diff
@@ -629,3 +632,33 @@ func (m *Manager) DeleteExpiredLinkAddresses(ctx context.Context, repository *gr
 	}
 	return itr.Err()
 }
+
+func (m *Manager) DeleteExpiredImports(ctx context.Context, repository *graveler.RepositoryRecord) error {
+	expiry := time.Now().Add(-ImportExpiryTime)
+	repoPartition := graveler.RepoPartition(repository)
+	key := []byte(graveler.ImportsPath(""))
+	options := kv.IteratorOptionsFrom([]byte(""))
+	itr, err := kv.NewPrimaryIterator(ctx, m.kvStoreLimited, (&graveler.ImportStatusData{}).ProtoReflect().Type(), repoPartition, key, options)
+	if err != nil {
+		return err
```

Comment:
Can we wrap the error in some identifier?  I worry that it might appear _really_ weird without context, and I will not be able to understand it when I see it in a log.

Reply:
Done